src/monitor/version_compat.h

```diff
@@ -14,38 +14,11 @@
 
 #include "postgres.h"
 
-/* we support Postgres versions 10, 11, 12, 13, 14, 15, and 16. */
-#if (PG_VERSION_NUM < 100000 || PG_VERSION_NUM >= 170000)
+/* we support Postgres versions 13, 14, 15, 16, and 17. */
+#if (PG_VERSION_NUM < 130000 || PG_VERSION_NUM >= 180000)
```

Comment:
Let's allow for building with older Postgres versions. Make it easy for people to tinker with it, unless it's a direct impediment to progress.

Reply:
In that case I'll drop the whole block.